Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```diff
@@ -94,7 +94,7 @@
       tank_slot:
```

Comment:
just change this to `GasTankSlot` instead of overriding the datafield name in c# imo